Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -880,6 +880,23 @@ internal string GetSerializedConfiguration()
             return JsonConvert.SerializeObject(this);
         }
 
+        private IConsistencyValidator consistencyValidator { get; set; }
```

Comment:
make this a private field with a default value.
```suggestion
        private IConsistencyValidator consistencyValidator = DefaultConsistencyValidator.Singleton;
```

Reply:
outdated